### PR TITLE
Feat alert

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Completed alert component

https://user-images.githubusercontent.com/88201664/152530949-7a60d3d5-f61a-4cf7-9d9d-08807b91b335.mp4

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#alert-close {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+#alert-close button {
+    margin-left: auto;
+    font-size: var(--size-10);
+    color: #970707;
```
Comment (GuptaEesh) [inline components/alert.css]: you can use css variables here for color

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+.alert-danger {
+    background-color: #ff000059;
+    color: #970707;
+}
+
+.alert-warning {
+    background-color: #fbff0050;
+    color: #ff9102;
+}
```
Comment (Hamdrive) [inline components/alert.css]: Prefer to use CSS variables which are predefined with these colors

File: docs/docs.html [hunk 3]
```diff
@@ -101,63 +101,85 @@ <h3 class="px-md py-sm">Components</h3>
...
+                <small>Example - Dismiss</small>
             </a>
         </nav>
     </main>
 </body>
 
 <script src="/docs/collapseMenu.js"></script>
-
+<script src="/components/js/alert.js"></script>
 </html>
```
Comment (Hamdrive) [inline docs/docs.html]: Add new empty line

**Review by Hamdrive — COMMENTED** [review]
Good work done, suggestions given

**Pull request merged**